src/http-header-glue.c

```diff
@@ -1452,6 +1452,9 @@ int http_cgi_headers (server *srv, connection *con, http_cgi_opts *opts, http_cg
     rc |= cb(vdata, CONST_STR_LEN("GATEWAY_INTERFACE"),
                     CONST_STR_LEN("CGI/1.1"));
 
+    rc |= cb(vdata, CONST_STR_LEN("REQUEST_SCHEME"),
+                    CONST_BUF_LEN(con->uri.scheme));
+
```

Comment:
@gstrauss, when you wrote _"Please move the code in http-header-glue.c to be right above where HTTPS variable is conditionally set"_ did you mean **_without_** inserting a blank line here?
(If so, I'll remove that blank line, of course.)